test/unit/dynamic-bootstrap-specs.js

```diff
@@ -44,7 +44,7 @@ describe('dynamic bootstrap', function () {
     rimraf(process.env.APPIUM_BOOTSTRAP_DIR)
 
       // first call: should create new bootstrap file
-      .then(function () { return prepareBootstrap(); })
+      .then(async function () { return await prepareBootstrap(); })
```

Comment:
agree with isaac. but also, these tests should be converted to use mochawait. didn't isaac already convert the tests too? i'm not sure what the relationship is between this PR and his.